gateway-test/src/test/java/org/apache/knox/gateway/AmbariServiceDefinitionTest.java

```diff
@@ -114,6 +114,12 @@ public static void setupGateway() throws Exception {
     File deployDir = new File( config.getGatewayDeploymentDir() );
     deployDir.mkdirs();
 
+    File descDir = new File( config.getGatewayDescriptorsDir() );
```

Comment:
This is out-of-scope of this JIRA so I filed a new one: https://issues.apache.org/jira/browse/KNOX-2309

Reply:
I had to make these changes to get the tests to pass. It is a side-effect requirement of the changes in this PR.